taxi/forms.py

```diff
@@ -50,3 +50,10 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
```

Comment:
The error message here states 'Last 5 characters should be digits', but the code checks for the last 5 characters to be digits. Ensure that the error message accurately reflects the validation logic.